modules/installation-network-user-infra.adoc

```diff
@@ -143,6 +158,37 @@ balancer after the bootstrap machine initializes the cluster control plane.
 |
 |Machine Config server
 
+|===
++
+[NOTE]
+====
+The load balancer should be configured to take a maximum of 30 seconds from the
```

Comment:
Is it _should be_ or _must_?

Reply:
This is a recommended time frame; so I think ***must*** would be too strong, since users could technically configure it to a different amount.

Reply:
It *must* be configured with a value <=30s. 20s is also fine. 40s is unsupportable.